release/release_gold.sh

```diff
@@ -36,9 +36,22 @@ function check_usage {
 	LIFERAY_COMMON_LOG_DIR="${_PROMOTION_DIR%/*}"
 }
 
+function check_supported_versions {
+	version="${LIFERAY_RELEASE_VERSION%.*}"
+
+	if [ -z $(grep "${version}" "${_RELEASE_ROOT_DIR}"/supported-"${LIFERAY_RELEASE_PRODUCT_NAME}"-versions.txt) ]
+	then
+		lc_log INFO "Version ${version} is not in the supported-${LIFERAY_RELEASE_PRODUCT_NAME}-versions.txt"
```

Comment:
@Wykenz, BChan prefers something like: `Unable to find ${version} in supported-${LIFERAY_RELEASE_PRODUCT_NAME}-versions.txt`.

Please see https://liferay.atlassian.net/wiki/spaces/ENGR/pages/1511752578/Wordsmith+conventions+in+Liferay#%5BinlineExtension%5D%22Unable-to%22%2C-not-%22Error%22